toolchain/check/testdata/interface/no_prelude/fail_lookup_undefined.carbon

```diff
@@ -20,25 +20,29 @@ interface Undefined;
 fn Undefined.F();
 
 fn Test() {
-  // CHECK:STDERR: fail_lookup_undefined.carbon:[[@LINE+7]]:3: error: member access into undefined interface `Undefined` [QualifiedExprInUndefinedInterfaceScope]
+  // CHECK:STDERR: fail_lookup_undefined.carbon:[[@LINE+11]]:3: error: member access into undefined interface Undefined [ResolveFacetTypeWithUndefinedInterface]
   // CHECK:STDERR:   Undefined.G();
   // CHECK:STDERR:   ^~~~~~~~~~~
   // CHECK:STDERR: fail_lookup_undefined.carbon:[[@LINE-15]]:1: note: interface was forward declared here [InterfaceForwardDeclaredHere]
   // CHECK:STDERR: interface Undefined;
   // CHECK:STDERR: ^~~~~~~~~~~~~~~~~~~~
   // CHECK:STDERR:
+  // CHECK:STDERR: fail_lookup_undefined.carbon:[[@LINE+4]]:3: error: member name `G` not found [MemberNameNotFound]
+  // CHECK:STDERR:   Undefined.G();
+  // CHECK:STDERR:   ^~~~~~~~~~~
+  // CHECK:STDERR:
```

Comment:
This looks like a regression. I think this means we're not adding an invalid scope in the case where facet type completion fails any more.